src/util/ReactUtils.ts

```diff
@@ -320,31 +320,26 @@ export const isSingleChildEqual = (nextChild: React.ReactElement, prevChild: Rea
 };
 
 export const renderByOrder = (children: React.ReactElement[], renderMap: any) => {
-  let elements: React.ReactElement[] = [];
+  const elements: React.ReactElement[] = [];
   const record: any = {};
 
   Children.forEach(children, (child, index) => {
-    if (child && isSvgElement(child)) {
+    if (isSvgElement(child)) {
       elements.push(child);
-    } else if (child && renderMap[getDisplayName(child.type)]) {
+    } else if (child) {
       const displayName = getDisplayName(child.type);
-      const { handler, once } = renderMap[displayName];
+      const { handler, once } = renderMap[displayName] || {};
 
-      if ((once && !record[displayName]) || !once) {
+      if (handler && (!once || !record[displayName])) {
         const results = handler(child, displayName, index);
 
-        if (_.isArray(results)) {
-          elements = [elements, ...results];
-        } else {
```

Comment:
It looked a little weird. :thinking: 
It should have been: `elements = [...elements, ...results]`?